docs/tutorials/configure/pet-photographer.md

```diff
@@ -0,0 +1,266 @@
+
+```{class="command-line" data-prompt="$"}
+realpath colorfilter 
+```
+
+{{< /alert >}}
+
+Read [this guide](https://docs.viam.com/extend/modular-resources/configure/#configure-a-local-module) for more information on configuring a local module.
+
+### Set up camera
+
+Navigate to your robot's page on the app and click on the **Config** tab.
+
+Add your robot's camera as a component by clicking **Create component** in lower-left corner of the page and typing in 'webcam' or specific the model you're using.
+
+![A photo of the webcam component named 'cam'](/tutorials/pet-photographer/webcam-component.png)
+
+For more information about the Camera component, you can refer to [this page](/components/camera/).
+
+## Add services
+
+After you've finished setting up, add a [vision service](/services/vision/detection) for color detection and a [data management service](/services/data/) for storing your filtered images.
+
+### Vision service to detect color
+
+This tutorial uses the color of my dogs collar, `#43A1D0` or `rgb(67, 161, 208)` (blue).
+
+**Hex color #43A1D0**: {{<imgproc src="/tutorials/pet-photographer/43a1d0.png" resize="90x" declaredimensions=true alt="A color swatch for the color of example subject's collar">}}
+
+Navigate to your robot's **Config** tab on the [Viam app](https://app.viam.com/robots) and configure your [vision service color detector](/services/vision/detection/):
+
+{{< tabs >}}
+{{% tab name="Builder" %}}
+
+1. Click the **Services** subtab and click **Create service** in the lower-left corner.
+
+1. Select the `Vision` type, then select the `Color Detector` model.
+
+1. Enter `my_color_detector` as the name for your detector and click **Create**.
+
+1. In the vision service panel, click the color selection box to set the color to be detected.
+   For this tutorial, set the color to the color of your pet, or use a blue/green collar or ribbon to increase the precision of your filter.
+
+1. Then, set **Hue Tolerance** to `0.06` and **Segment Size px** to `100`.
+
+Your configuration should look like the following:
+
+![The vision service configuration panel showing the color set to blue, the hue tolerance set to 0.06, and the segment size set to 100.](/tutorials/pet-photographer/vision-service.png)
+
+For more detailed information see [Configure a color detector](/services/vision/detection/#configure-a-color_detector).
+
+{{% /tab %}}
+{{% tab name="JSON Template" %}}
+
+Add the vision service object to the services array in your rover’s raw JSON configuration:
+
+```json {class="line-numbers linkable-line-numbers"}
+"services": [
+  {
+    "name": "my_color_detector",
+    "type": "vision",
+    "model": "color_detector",
+    "attributes": {
+      "segment_size_px": 100,
+      "detect_color": "#43a1d0",
+      "hue_tolerance_pct": 0.06
+    }
+  },
+  ... // Other services
+]
```

Comment:
The danger is that they will otherwise copy paste the services object as a whole
```suggestion
  {
    "name": "my_color_detector",
    "type": "vision",
    "model": "color_detector",
    "attributes": {
      "segment_size_px": 100,
      "detect_color": "#43a1d0",
      "hue_tolerance_pct": 0.06
    }
  },
  ... // Other services
```